cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -918,6 +918,10 @@ private boolean applyAndRun(CliArguments arguments, Commandlet cmd) {
           debug(getMessageIdeHomeFound());
         }
       }
+      if (getGitContext().isRepositoryUpdateAvailable(getSettingsPath()))
+        info("Updates are available for the settings repository. If you want to pull the latest changes, call ide update.");
+      else
```

Comment:
can you check your IDE setup?
You should call `ide update` to get your IntelliJ settings up-to-date so that code is nicely autoformatted, curly braces added, etc.